packages/blob/src/create-folder.ts

```diff
@@ -0,0 +1,44 @@
+import { requestApi } from './api';
+import type { BlobCommandOptions } from './helpers';
+import { putOptionHeaderMap, type PutBlobApiResponse } from './put-helpers';
+
+export interface CreateFolderResult {
+  pathname: string;
+  /**
+   * URL of created folder. There is not content at this URL.
+   * Use this URL to delete the folder, just like you would delete a file.
+   */
+  url: string;
+}
+
+/**
+ * Creates a folder in your store. Vercel Blob has no real concept of folders, our file browser on Vercel.com displays folders based on the presence of trailing slashes in the pathname. Unless you are building a file browser system, you probably don't need to use this method.
```

Comment:
should we warn here, that folders will appear as blobs when you call `list()` ?https://github.com/vercel/storage/issues/640